actions/search_instances.go

```diff
@@ -56,12 +57,27 @@ func (s searcher) searchInstances(c gcloud.Client, args config.Args, matcher sto
 
 func (s searcher) formatInstances(insts []gcloud.Instance, args config.Args) error {
 	hostMapping := args.InstanceCmdArgs.HostMapping
+	ipOnly := args.InstanceCmdArgs.IPOnly
+
 	if hostMapping {
 		for _, ins := range insts {
 			fmt.Printf("%-16s  %s\n", ins.IP(), ins.Name)
 		}
 		return nil
 	}
+
+	if ipOnly {
+		output := ""
+		sep := ","
+		for _, ins := range insts {
+			output += ins.IP() + sep
+		}
+		output = strings.TrimRight(output, ",")
+		fmt.Printf("%s", output)
+		fmt.Printf("\n")
```

Comment:
we can get the ip list with our custom `type Instances []Instance` with IPs defined on the struct.
```
ips := Instances(insts).IPs()
strings.Join(ips, ",")
```